Rasa process starting
Rasa process starting
Rasa process starting
Rasa process starting
Rasa process starting
Rasa process starting
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
Looking for jobs to run
No jobs; waiting until a job is added
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Available web server routes: 
hello                                              OPTIONS, GET, HEAD             /
list_trackers                                      OPTIONS, GET, HEAD             /conversations
execute_action                                     POST, OPTIONS                  /conversations/[sender_id]/execute
log_message                                        POST, OPTIONS                  /conversations/[sender_id]/messages
predict                                            POST, OPTIONS                  /conversations/[sender_id]/predict
respond                                            POST, OPTIONS, GET, HEAD       /conversations/[sender_id]/respond
retrieve_story                                     OPTIONS, GET, HEAD             /conversations/[sender_id]/story
retrieve_tracker                                   OPTIONS, GET, HEAD             /conversations/[sender_id]/tracker
replace_events                                     OPTIONS, PUT                   /conversations/[sender_id]/tracker/events
get_domain                                         OPTIONS, GET, HEAD             /domain
evaluate_stories                                   POST, OPTIONS                  /evaluate
continue_training                                  POST, OPTIONS                  /finetune
load_model                                         POST, OPTIONS                  /model
parse                                              POST, OPTIONS                  /parse
tracker_predict                                    POST, OPTIONS                  /predict
static                                             OPTIONS, GET, HEAD             /static/[filename]
status                                             OPTIONS, GET, HEAD             /status
version                                            OPTIONS, GET, HEAD             /version
custom_webhook_CmdlineInput.health                 OPTIONS, GET, HEAD             /webhooks/rest/
custom_webhook_CmdlineInput.receive                POST, OPTIONS                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5001
Creating a new tracker for id 'default'.
Received user message 'hello' with intent '{'name': 'greet', 'confidence': 0.7600140671252895}' and entities '[]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	location: None
Current tracker state [None, None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '8'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_greet' with prob 1.00.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'tell me the weather' with intent '{'name': 'inform', 'confidence': 0.8385695444952107}' and entities '[]'
Logged UserUtterance - tracker now has 6 events
Current slot values: 
	location: None
Current tracker state [None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '10'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_ask_location' with prob 1.00.
Action 'utter_ask_location' ended with events '[]'
Bot utterance 'BotUttered(text: In what location?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'in italy' with intent '{'name': 'inform', 'confidence': 0.8753292785202255}' and entities '[{'start': 3, 'end': 8, 'value': 'italy', 'entity': 'location', 'confidence': 0.9223207185313498, 'extractor': 'CRFEntityExtractor'}]'
Logged UserUtterance - tracker now has 11 events
Current slot values: 
	location: italy
Current tracker state [{'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0}]
There is a memorised next action '11'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_weather' with prob 1.00.
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
The model predicted the custom action 'action_weather' but you didn't configure an endpoint to run this custom action. Please take a look at the docs and set an endpoint configuration. https://legacy-docs.rasa.com/docs/core/customactions/
Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\processor.py", line 345, in _run_action
    events = action.run(dispatcher, tracker, self.domain)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\actions\action.py", line 341, in run
    "".format(self.name(), DOCS_BASE_URL))
Exception: The model predicted the custom action 'action_weather' but you didn't configure an endpoint to run this custom action. Please take a look at the docs and set an endpoint configuration. https://legacy-docs.rasa.com/docs/core/customactions/
Action 'action_weather' ended with events '[]'
Current tracker state [{'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0}, {'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_action_weather': 1.0, 'entity_location': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'e' with intent '{'name': 'inform', 'confidence': 0.6351787303462773}' and entities '[]'
Logged UserUtterance - tracker now has 14 events
Current slot values: 
	location: italy
Current tracker state [{'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0}, {'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_action_weather': 1.0, 'entity_location': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'utter_goodbye' with prob 0.92.
Action 'utter_goodbye' ended with events '[]'
Bot utterance 'BotUttered(text: Talk to you later., data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0}, {'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_action_weather': 1.0, 'entity_location': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}, {'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_utter_goodbye': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_listen' with prob 0.91.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'stop' with intent '{'name': 'inform', 'confidence': 0.5901361433524825}' and entities '[]'
Logged UserUtterance - tracker now has 18 events
Current slot values: 
	location: italy
Current tracker state [{'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0}, {'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_action_weather': 1.0, 'entity_location': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}, {'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_utter_goodbye': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'utter_greet' with prob 0.54.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_action_weather': 1.0, 'entity_location': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}, {'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_utter_goodbye': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}, {'prev_utter_greet': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_listen' with prob 0.98.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'exit' with intent '{'name': 'goodbye', 'confidence': 0.4723445935573423}' and entities '[]'
Logged UserUtterance - tracker now has 22 events
Current slot values: 
	location: italy
Current tracker state [{'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}, {'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_utter_goodbye': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}, {'prev_utter_greet': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_goodbye': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'utter_greet' with prob 0.36.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_utter_goodbye': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}, {'prev_utter_greet': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0}, {'prev_action_listen': 1.0, 'slot_location_0': 1.0, 'intent_goodbye': 1.0}, {'prev_utter_greet': 1.0, 'slot_location_0': 1.0, 'intent_goodbye': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_listen' with prob 0.99.
Action 'action_listen' ended with events '[]'
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5001
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5001
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
Looking for jobs to run
No jobs; waiting until a job is added
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Available web server routes: 
hello                                              OPTIONS, HEAD, GET             /
list_trackers                                      OPTIONS, HEAD, GET             /conversations
execute_action                                     OPTIONS, POST                  /conversations/[sender_id]/execute
log_message                                        OPTIONS, POST                  /conversations/[sender_id]/messages
predict                                            OPTIONS, POST                  /conversations/[sender_id]/predict
respond                                            OPTIONS, HEAD, GET, POST       /conversations/[sender_id]/respond
retrieve_story                                     OPTIONS, HEAD, GET             /conversations/[sender_id]/story
retrieve_tracker                                   OPTIONS, HEAD, GET             /conversations/[sender_id]/tracker
replace_events                                     OPTIONS, PUT                   /conversations/[sender_id]/tracker/events
get_domain                                         OPTIONS, HEAD, GET             /domain
evaluate_stories                                   OPTIONS, POST                  /evaluate
continue_training                                  OPTIONS, POST                  /finetune
load_model                                         OPTIONS, POST                  /model
parse                                              OPTIONS, POST                  /parse
tracker_predict                                    OPTIONS, POST                  /predict
static                                             OPTIONS, HEAD, GET             /static/[filename]
status                                             OPTIONS, HEAD, GET             /status
version                                            OPTIONS, HEAD, GET             /version
custom_webhook_CmdlineInput.health                 OPTIONS, HEAD, GET             /webhooks/rest/
custom_webhook_CmdlineInput.receive                OPTIONS, POST                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5001
Creating a new tracker for id 'default'.
Received user message 'hello' with intent '{'name': 'greet', 'confidence': 0.7600140671252895}' and entities '[]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	location: None
Current tracker state [None, None, None, {}, {'prev_action_listen': 1.0, 'intent_greet': 1.0}]
There is a memorised next action '8'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_greet' with prob 1.00.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, {}, {'prev_action_listen': 1.0, 'intent_greet': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'weather' with intent '{'name': 'inform', 'confidence': 0.9799579792302853}' and entities '[]'
Logged UserUtterance - tracker now has 6 events
Current slot values: 
	location: None
Current tracker state [None, {}, {'prev_action_listen': 1.0, 'intent_greet': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '10'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_ask_location' with prob 1.00.
Action 'utter_ask_location' ended with events '[]'
Bot utterance 'BotUttered(text: In what location?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{}, {'prev_action_listen': 1.0, 'intent_greet': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'in Tunisia' with intent '{'name': 'inform', 'confidence': 0.8753292785202255}' and entities '[{'start': 3, 'end': 10, 'value': 'tunisia', 'entity': 'location', 'confidence': 0.8286146105114172, 'extractor': 'CRFEntityExtractor'}]'
Logged UserUtterance - tracker now has 11 events
Current slot values: 
	location: tunisia
Current tracker state [{'prev_action_listen': 1.0, 'intent_greet': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'entity_location': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '11'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_weather' with prob 1.00.
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
The model predicted the custom action 'action_weather' but you didn't configure an endpoint to run this custom action. Please take a look at the docs and set an endpoint configuration. https://legacy-docs.rasa.com/docs/core/customactions/
Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\processor.py", line 345, in _run_action
    events = action.run(dispatcher, tracker, self.domain)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\actions\action.py", line 341, in run
    "".format(self.name(), DOCS_BASE_URL))
Exception: The model predicted the custom action 'action_weather' but you didn't configure an endpoint to run this custom action. Please take a look at the docs and set an endpoint configuration. https://legacy-docs.rasa.com/docs/core/customactions/
Action 'action_weather' ended with events '[]'
Current tracker state [{'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'entity_location': 1.0, 'intent_inform': 1.0}, {'slot_location_0': 1.0, 'entity_location': 1.0, 'intent_inform': 1.0, 'prev_action_weather': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
Looking for jobs to run
No jobs; waiting until a job is added
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Available web server routes: 
hello                                              HEAD, GET, OPTIONS             /
list_trackers                                      HEAD, GET, OPTIONS             /conversations
execute_action                                     POST, OPTIONS                  /conversations/[sender_id]/execute
log_message                                        POST, OPTIONS                  /conversations/[sender_id]/messages
predict                                            POST, OPTIONS                  /conversations/[sender_id]/predict
respond                                            HEAD, POST, GET, OPTIONS       /conversations/[sender_id]/respond
retrieve_story                                     HEAD, GET, OPTIONS             /conversations/[sender_id]/story
retrieve_tracker                                   HEAD, GET, OPTIONS             /conversations/[sender_id]/tracker
replace_events                                     PUT, OPTIONS                   /conversations/[sender_id]/tracker/events
get_domain                                         HEAD, GET, OPTIONS             /domain
evaluate_stories                                   POST, OPTIONS                  /evaluate
continue_training                                  POST, OPTIONS                  /finetune
load_model                                         POST, OPTIONS                  /model
parse                                              POST, OPTIONS                  /parse
tracker_predict                                    POST, OPTIONS                  /predict
static                                             HEAD, GET, OPTIONS             /static/[filename]
status                                             HEAD, GET, OPTIONS             /status
version                                            HEAD, GET, OPTIONS             /version
custom_webhook_CmdlineInput.health                 HEAD, GET, OPTIONS             /webhooks/rest/
custom_webhook_CmdlineInput.receive                POST, OPTIONS                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5001
Creating a new tracker for id 'default'.
Received user message 'hello' with intent '{'name': 'greet', 'confidence': 0.7600140671252895}' and entities '[]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	location: None
Current tracker state [None, None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '8'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_greet' with prob 1.00.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'weather' with intent '{'name': 'inform', 'confidence': 0.9799579792302853}' and entities '[]'
Logged UserUtterance - tracker now has 6 events
Current slot values: 
	location: None
Current tracker state [None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '10'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_ask_location' with prob 1.00.
Action 'utter_ask_location' ended with events '[]'
Bot utterance 'BotUttered(text: In what location?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_ask_location': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'in Tunisia' with intent '{'name': 'inform', 'confidence': 0.8753292785202255}' and entities '[{'start': 3, 'end': 10, 'value': 'tunisia', 'entity': 'location', 'confidence': 0.8286146105114172, 'extractor': 'CRFEntityExtractor'}]'
Logged UserUtterance - tracker now has 11 events
Current slot values: 
	location: tunisia
Current tracker state [{'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_ask_location': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0}]
There is a memorised next action '11'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_weather' with prob 1.00.
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
The model predicted the custom action 'action_weather' but you didn't configure an endpoint to run this custom action. Please take a look at the docs and set an endpoint configuration. https://legacy-docs.rasa.com/docs/core/customactions/
Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\processor.py", line 345, in _run_action
    events = action.run(dispatcher, tracker, self.domain)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\actions\action.py", line 341, in run
    "".format(self.name(), DOCS_BASE_URL))
Exception: The model predicted the custom action 'action_weather' but you didn't configure an endpoint to run this custom action. Please take a look at the docs and set an endpoint configuration. https://legacy-docs.rasa.com/docs/core/customactions/
Action 'action_weather' ended with events '[]'
Current tracker state [{'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_ask_location': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0}, {'slot_location_0': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'prev_action_weather': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
Looking for jobs to run
No jobs; waiting until a job is added
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Available web server routes: 
hello                                              HEAD, GET, OPTIONS             /
list_trackers                                      HEAD, GET, OPTIONS             /conversations
execute_action                                     POST, OPTIONS                  /conversations/[sender_id]/execute
log_message                                        POST, OPTIONS                  /conversations/[sender_id]/messages
predict                                            POST, OPTIONS                  /conversations/[sender_id]/predict
respond                                            HEAD, POST, GET, OPTIONS       /conversations/[sender_id]/respond
retrieve_story                                     HEAD, GET, OPTIONS             /conversations/[sender_id]/story
retrieve_tracker                                   HEAD, GET, OPTIONS             /conversations/[sender_id]/tracker
replace_events                                     PUT, OPTIONS                   /conversations/[sender_id]/tracker/events
get_domain                                         HEAD, GET, OPTIONS             /domain
evaluate_stories                                   POST, OPTIONS                  /evaluate
continue_training                                  POST, OPTIONS                  /finetune
load_model                                         POST, OPTIONS                  /model
parse                                              POST, OPTIONS                  /parse
tracker_predict                                    POST, OPTIONS                  /predict
static                                             HEAD, GET, OPTIONS             /static/[filename]
status                                             HEAD, GET, OPTIONS             /status
version                                            HEAD, GET, OPTIONS             /version
custom_webhook_CmdlineInput.health                 HEAD, GET, OPTIONS             /webhooks/rest/
custom_webhook_CmdlineInput.receive                POST, OPTIONS                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5001
Creating a new tracker for id 'default'.
Received user message 'hello' with intent '{'name': 'greet', 'confidence': 0.7600140671252895}' and entities '[]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	location: None
Current tracker state [None, None, None, {}, {'prev_action_listen': 1.0, 'intent_greet': 1.0}]
There is a memorised next action '8'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_greet' with prob 1.00.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, {}, {'prev_action_listen': 1.0, 'intent_greet': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'weather' with intent '{'name': 'inform', 'confidence': 0.9799579792302853}' and entities '[]'
Logged UserUtterance - tracker now has 6 events
Current slot values: 
	location: None
Current tracker state [None, {}, {'prev_action_listen': 1.0, 'intent_greet': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '10'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_ask_location' with prob 1.00.
Action 'utter_ask_location' ended with events '[]'
Bot utterance 'BotUttered(text: In what location?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{}, {'prev_action_listen': 1.0, 'intent_greet': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'in Tunisia' with intent '{'name': 'inform', 'confidence': 0.8753292785202255}' and entities '[{'start': 3, 'end': 10, 'value': 'tunisia', 'entity': 'location', 'confidence': 0.8286146105114172, 'extractor': 'CRFEntityExtractor'}]'
Logged UserUtterance - tracker now has 11 events
Current slot values: 
	location: tunisia
Current tracker state [{'prev_action_listen': 1.0, 'intent_greet': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '11'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_weather' with prob 1.00.
Calling action endpoint to run action 'action_weather'.
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
HTTPConnectionPool(host='localhost', port=5001): Read timed out. (read timeout=300)
Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 1331, in getresponse
    response.begin()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 258, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\socket.py", line 586, in readinto
    return self._sock.recv_into(b)
socket.timeout: timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\adapters.py", line 449, in send
    timeout=timeout
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 641, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\util\retry.py", line 368, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\packages\six.py", line 686, in reraise
    raise value
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 603, in urlopen
    chunked=chunked)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 389, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 307, in _raise_timeout
    raise ReadTimeoutError(self, url, "Read timed out. (read timeout=%s)" % timeout_value)
urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=5001): Read timed out. (read timeout=300)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\processor.py", line 345, in _run_action
    events = action.run(dispatcher, tracker, self.domain)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\actions\action.py", line 347, in run
    json=json, method="post", timeout=DEFAULT_REQUEST_TIMEOUT)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\utils.py", line 702, in request
    **kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPConnectionPool(host='localhost', port=5001): Read timed out. (read timeout=300)
Action 'action_weather' ended with events '[]'
Current tracker state [{'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'entity_location': 1.0, 'slot_location_0': 1.0, 'intent_inform': 1.0, 'prev_action_weather': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
Looking for jobs to run
No jobs; waiting until a job is added
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Available web server routes: 
hello                                              HEAD, GET, OPTIONS             /
list_trackers                                      HEAD, GET, OPTIONS             /conversations
execute_action                                     POST, OPTIONS                  /conversations/[sender_id]/execute
log_message                                        POST, OPTIONS                  /conversations/[sender_id]/messages
predict                                            POST, OPTIONS                  /conversations/[sender_id]/predict
respond                                            HEAD, GET, OPTIONS, POST       /conversations/[sender_id]/respond
retrieve_story                                     HEAD, GET, OPTIONS             /conversations/[sender_id]/story
retrieve_tracker                                   HEAD, GET, OPTIONS             /conversations/[sender_id]/tracker
replace_events                                     PUT, OPTIONS                   /conversations/[sender_id]/tracker/events
get_domain                                         HEAD, GET, OPTIONS             /domain
evaluate_stories                                   POST, OPTIONS                  /evaluate
continue_training                                  POST, OPTIONS                  /finetune
load_model                                         POST, OPTIONS                  /model
parse                                              POST, OPTIONS                  /parse
tracker_predict                                    POST, OPTIONS                  /predict
static                                             HEAD, GET, OPTIONS             /static/[filename]
status                                             HEAD, GET, OPTIONS             /status
version                                            HEAD, GET, OPTIONS             /version
custom_webhook_CmdlineInput.health                 HEAD, GET, OPTIONS             /webhooks/rest/
custom_webhook_CmdlineInput.receive                POST, OPTIONS                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5001
Creating a new tracker for id 'default'.
Received user message 'hello' with intent '{'name': 'greet', 'confidence': 0.7600140671252895}' and entities '[]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	location: None
Current tracker state [None, None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '8'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_greet' with prob 1.00.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'weather' with intent '{'name': 'inform', 'confidence': 0.9799579792302853}' and entities '[]'
Logged UserUtterance - tracker now has 6 events
Current slot values: 
	location: None
Current tracker state [None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '10'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_ask_location' with prob 1.00.
Action 'utter_ask_location' ended with events '[]'
Bot utterance 'BotUttered(text: In what location?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_ask_location': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'in Tunisia' with intent '{'name': 'inform', 'confidence': 0.8753292785202255}' and entities '[{'start': 3, 'end': 10, 'value': 'tunisia', 'entity': 'location', 'confidence': 0.8286146105114172, 'extractor': 'CRFEntityExtractor'}]'
Logged UserUtterance - tracker now has 11 events
Current slot values: 
	location: tunisia
Current tracker state [{'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_ask_location': 1.0}, {'entity_location': 1.0, 'intent_inform': 1.0, 'prev_action_listen': 1.0, 'slot_location_0': 1.0}]
There is a memorised next action '11'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_weather' with prob 1.00.
Calling action endpoint to run action 'action_weather'.
Failed to run custom action 'action_weather'. Couldn't connect to the server at 'http://localhost:5055/webhook'. Is the server running? Error: HTTPConnectionPool(host='localhost', port=5055): Max retries exceeded with url: /webhook (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002956A42A898>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée',))
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Failed to execute custom action.
Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connection.py", line 160, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\util\connection.py", line 80, in create_connection
    raise err
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\util\connection.py", line 70, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 603, in urlopen
    chunked=chunked)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 355, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 1239, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 1285, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 1234, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 1026, in _send_output
    self.send(msg)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 964, in send
    self.connect()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connection.py", line 183, in connect
    conn = self._new_conn()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e)
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x000002956A42A898>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\adapters.py", line 449, in send
    timeout=timeout
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 641, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\util\retry.py", line 399, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=5055): Max retries exceeded with url: /webhook (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002956A42A898>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\actions\action.py", line 347, in run
    json=json, method="post", timeout=DEFAULT_REQUEST_TIMEOUT)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\utils.py", line 702, in request
    **kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=5055): Max retries exceeded with url: /webhook (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002956A42A898>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\processor.py", line 345, in _run_action
    events = action.run(dispatcher, tracker, self.domain)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\actions\action.py", line 368, in run
    raise Exception("Failed to execute custom action.")
Exception: Failed to execute custom action.
Action 'action_weather' ended with events '[]'
Current tracker state [{'prev_utter_greet': 1.0, 'intent_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_ask_location': 1.0}, {'entity_location': 1.0, 'intent_inform': 1.0, 'prev_action_listen': 1.0, 'slot_location_0': 1.0}, {'entity_location': 1.0, 'intent_inform': 1.0, 'prev_action_weather': 1.0, 'slot_location_0': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
Looking for jobs to run
No jobs; waiting until a job is added
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Available web server routes: 
hello                                              GET, HEAD, OPTIONS             /
list_trackers                                      GET, HEAD, OPTIONS             /conversations
execute_action                                     OPTIONS, POST                  /conversations/[sender_id]/execute
log_message                                        OPTIONS, POST                  /conversations/[sender_id]/messages
predict                                            OPTIONS, POST                  /conversations/[sender_id]/predict
respond                                            GET, HEAD, OPTIONS, POST       /conversations/[sender_id]/respond
retrieve_story                                     GET, HEAD, OPTIONS             /conversations/[sender_id]/story
retrieve_tracker                                   GET, HEAD, OPTIONS             /conversations/[sender_id]/tracker
replace_events                                     OPTIONS, PUT                   /conversations/[sender_id]/tracker/events
get_domain                                         GET, HEAD, OPTIONS             /domain
evaluate_stories                                   OPTIONS, POST                  /evaluate
continue_training                                  OPTIONS, POST                  /finetune
load_model                                         OPTIONS, POST                  /model
parse                                              OPTIONS, POST                  /parse
tracker_predict                                    OPTIONS, POST                  /predict
static                                             GET, HEAD, OPTIONS             /static/[filename]
status                                             GET, HEAD, OPTIONS             /status
version                                            GET, HEAD, OPTIONS             /version
custom_webhook_CmdlineInput.health                 GET, HEAD, OPTIONS             /webhooks/rest/
custom_webhook_CmdlineInput.receive                OPTIONS, POST                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5001
Creating a new tracker for id 'default'.
Received user message 'hello' with intent '{'name': 'greet', 'confidence': 0.7600140671252895}' and entities '[]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	location: None
Current tracker state [None, None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '8'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_greet' with prob 1.00.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'weather' with intent '{'name': 'inform', 'confidence': 0.9799579792302853}' and entities '[]'
Logged UserUtterance - tracker now has 6 events
Current slot values: 
	location: None
Current tracker state [None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '10'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_ask_location' with prob 1.00.
Action 'utter_ask_location' ended with events '[]'
Bot utterance 'BotUttered(text: In what location?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_utter_ask_location': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'tell me the weather' with intent '{'name': 'inform', 'confidence': 0.8385695444952107}' and entities '[]'
Logged UserUtterance - tracker now has 10 events
Current slot values: 
	location: None
Current tracker state [{'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_utter_ask_location': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_weather' with prob 0.87.
Calling action endpoint to run action 'action_weather'.
Failed to run custom action 'action_weather'. Couldn't connect to the server at 'http://localhost:5001/webhook'. Is the server running? Error: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Failed to execute custom action.
Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 603, in urlopen
    chunked=chunked)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 1331, in getresponse
    response.begin()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 258, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\socket.py", line 586, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [WinError 10054] Une connexion existante a dû être fermée par l’hôte distant

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\adapters.py", line 449, in send
    timeout=timeout
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 641, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\util\retry.py", line 368, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\packages\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 603, in urlopen
    chunked=chunked)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 1331, in getresponse
    response.begin()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\http\client.py", line 258, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\socket.py", line 586, in readinto
    return self._sock.recv_into(b)
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\actions\action.py", line 347, in run
    json=json, method="post", timeout=DEFAULT_REQUEST_TIMEOUT)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\utils.py", line 702, in request
    **kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\requests\adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\processor.py", line 345, in _run_action
    events = action.run(dispatcher, tracker, self.domain)
  File "C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\rasa_core\actions\action.py", line 368, in run
    raise Exception("Failed to execute custom action.")
Exception: Failed to execute custom action.
Action 'action_weather' ended with events '[]'
Current tracker state [{'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_utter_ask_location': 1.0}, {'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_weather': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_listen' with prob 0.90.
Action 'action_listen' ended with events '[]'
Rasa process starting
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5001
Failed to run custom action 'action_weather'. Couldn't connect to the server at 'http://localhost:5001/webhook'. Is the server running? Error: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Rasa process starting
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
Looking for jobs to run
No jobs; waiting until a job is added
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Available web server routes: 
hello                                              GET, HEAD, OPTIONS             /
list_trackers                                      GET, HEAD, OPTIONS             /conversations
execute_action                                     OPTIONS, POST                  /conversations/[sender_id]/execute
log_message                                        OPTIONS, POST                  /conversations/[sender_id]/messages
predict                                            OPTIONS, POST                  /conversations/[sender_id]/predict
respond                                            GET, HEAD, OPTIONS, POST       /conversations/[sender_id]/respond
retrieve_story                                     GET, HEAD, OPTIONS             /conversations/[sender_id]/story
retrieve_tracker                                   GET, HEAD, OPTIONS             /conversations/[sender_id]/tracker
replace_events                                     PUT, OPTIONS                   /conversations/[sender_id]/tracker/events
get_domain                                         GET, HEAD, OPTIONS             /domain
evaluate_stories                                   OPTIONS, POST                  /evaluate
continue_training                                  OPTIONS, POST                  /finetune
load_model                                         OPTIONS, POST                  /model
parse                                              OPTIONS, POST                  /parse
tracker_predict                                    OPTIONS, POST                  /predict
static                                             GET, HEAD, OPTIONS             /static/[filename]
status                                             GET, HEAD, OPTIONS             /status
version                                            GET, HEAD, OPTIONS             /version
custom_webhook_CmdlineInput.health                 GET, HEAD, OPTIONS             /webhooks/rest/
custom_webhook_CmdlineInput.receive                OPTIONS, POST                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5005
Creating a new tracker for id 'default'.
Received user message 'hi' with intent '{'name': 'greet', 'confidence': 0.8699418559726987}' and entities '[]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	location: None
Current tracker state [None, None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '8'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_greet' with prob 1.00.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'weather' with intent '{'name': 'inform', 'confidence': 0.9799579792302853}' and entities '[]'
Logged UserUtterance - tracker now has 6 events
Current slot values: 
	location: None
Current tracker state [None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '10'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_ask_location' with prob 1.00.
Action 'utter_ask_location' ended with events '[]'
Bot utterance 'BotUttered(text: In what location?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'in Tunisia' with intent '{'name': 'inform', 'confidence': 0.8753292785202255}' and entities '[{'start': 3, 'end': 10, 'value': 'tunisia', 'entity': 'location', 'confidence': 0.8286146105114172, 'extractor': 'CRFEntityExtractor'}]'
Logged UserUtterance - tracker now has 11 events
Current slot values: 
	location: tunisia
Current tracker state [{'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '11'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_weather' with prob 1.00.
Calling action endpoint to run action 'action_weather'.
Action 'action_weather' ended with events '['SlotSet(key: location, value: tunisia)']'
Bot utterance 'BotUttered(text: it is currently Partly cloudy in Tunis at the moment the temperature is 31.0 degrees,
		the humidity is 36 and the wind speed is 12.5 mph, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'bye' with intent '{'name': 'goodbye', 'confidence': 0.8967853717352816}' and entities '[]'
Logged UserUtterance - tracker now has 16 events
Current slot values: 
	location: tunisia
Current tracker state [{'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_goodbye': 1.0}]
There is a memorised next action '9'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_goodbye' with prob 1.00.
Action 'utter_goodbye' ended with events '[]'
Bot utterance 'BotUttered(text: Talk to you later., data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_goodbye': 1.0, 'prev_utter_goodbye': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Creating a new tracker for id 'meyssa'.
Received user message 'in Italy' with intent '{'name': 'inform', 'confidence': 0.8753292785202255}' and entities '[{'start': 3, 'end': 8, 'value': 'italy', 'entity': 'location', 'confidence': 0.9223207185313498, 'extractor': 'CRFEntityExtractor'}]'
Logged UserUtterance - tracker now has 3 events
Current slot values: 
	location: italy
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Action 'action_listen' was predicted after a user message using policy_0_KerasPolicy. Predicting fallback action: action_default_fallback
Predicted next action using policy_1_FallbackPolicy
Predicted next action 'action_default_fallback' with prob 1.00.
Action 'action_default_fallback' ended with events '['UserUtteranceReverted()']'
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'meyssa'
Received user message 'hi' with intent '{'name': 'greet', 'confidence': 0.8699418559726987}' and entities '[]'
Logged UserUtterance - tracker now has 7 events
Current slot values: 
	location: None
Current tracker state [None, None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '8'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_greet' with prob 1.00.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'meyssa'
Received user message 'weather' with intent '{'name': 'inform', 'confidence': 0.9799579792302853}' and entities '[]'
Logged UserUtterance - tracker now has 11 events
Current slot values: 
	location: None
Current tracker state [None, {}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '10'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_ask_location' with prob 1.00.
Action 'utter_ask_location' ended with events '[]'
Bot utterance 'BotUttered(text: In what location?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{}, {'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'meyssa'
Received user message 'in Spain' with intent '{'name': 'inform', 'confidence': 0.8753292785202255}' and entities '[{'start': 3, 'end': 8, 'value': 'spain', 'entity': 'location', 'confidence': 0.7774977740103174, 'extractor': 'CRFEntityExtractor'}]'
Logged UserUtterance - tracker now has 16 events
Current slot values: 
	location: spain
Current tracker state [{'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}]
There is a memorised next action '11'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_weather' with prob 1.00.
Calling action endpoint to run action 'action_weather'.
Action 'action_weather' ended with events '['SlotSet(key: location, value: spain)']'
Bot utterance 'BotUttered(text: it is currently Sunny in Madrid at the moment the temperature is 29.0 degrees,
		the humidity is 25 and the wind speed is 4.3 mph, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'meyssa'
Received user message 'bye' with intent '{'name': 'goodbye', 'confidence': 0.8967853717352816}' and entities '[]'
Logged UserUtterance - tracker now has 21 events
Current slot values: 
	location: spain
Current tracker state [{'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_goodbye': 1.0}]
There is a memorised next action '9'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'utter_goodbye' with prob 1.00.
Action 'utter_goodbye' ended with events '[]'
Bot utterance 'BotUttered(text: Talk to you later., data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'prev_utter_ask_location': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_goodbye': 1.0, 'prev_utter_goodbye': 1.0}]
There is a memorised next action '0'
There is no active form
Predicted next action using policy_2_MemoizationPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'meyssa'
Received user message 'hi' with intent '{'name': 'greet', 'confidence': 0.8699418559726987}' and entities '[]'
Logged UserUtterance - tracker now has 25 events
Current slot values: 
	location: spain
Current tracker state [{'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_goodbye': 1.0, 'prev_utter_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'utter_greet' with prob 0.82.
Action 'utter_greet' ended with events '[]'
Bot utterance 'BotUttered(text: Hello! How can I help?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_goodbye': 1.0, 'prev_utter_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_utter_greet': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_listen' with prob 0.91.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'meyssa'
Received user message 'weather' with intent '{'name': 'inform', 'confidence': 0.9799579792302853}' and entities '[]'
Logged UserUtterance - tracker now has 29 events
Current slot values: 
	location: spain
Current tracker state [{'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_goodbye': 1.0, 'prev_utter_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'utter_ask_location' with prob 0.86.
Action 'utter_ask_location' ended with events '[]'
Bot utterance 'BotUttered(text: In what location?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'slot_location_0': 1.0, 'intent_goodbye': 1.0, 'prev_utter_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_utter_ask_location': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_listen' with prob 0.87.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'meyssa'
Received user message 'in Italy' with intent '{'name': 'inform', 'confidence': 0.8753292785202255}' and entities '[{'start': 3, 'end': 8, 'value': 'italy', 'entity': 'location', 'confidence': 0.9223207185313498, 'extractor': 'CRFEntityExtractor'}]'
Logged UserUtterance - tracker now has 34 events
Current slot values: 
	location: italy
Current tracker state [{'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_action_listen': 1.0}, {'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_utter_ask_location': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_weather' with prob 0.90.
Calling action endpoint to run action 'action_weather'.
Action 'action_weather' ended with events '['SlotSet(key: location, value: italy)']'
Bot utterance 'BotUttered(text: it is currently Sunny in Rome at the moment the temperature is 28.0 degrees,
		the humidity is 66 and the wind speed is 0.0 mph, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'slot_location_0': 1.0, 'intent_greet': 1.0, 'prev_utter_greet': 1.0}, {'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_utter_ask_location': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_listen' with prob 0.94.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'meyssa'
Received user message 'bye' with intent '{'name': 'goodbye', 'confidence': 0.8967853717352816}' and entities '[]'
Logged UserUtterance - tracker now has 39 events
Current slot values: 
	location: italy
Current tracker state [{'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_utter_ask_location': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_goodbye': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'utter_goodbye' with prob 0.92.
Action 'utter_goodbye' ended with events '[]'
Bot utterance 'BotUttered(text: Talk to you later., data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_inform': 1.0, 'slot_location_0': 1.0, 'prev_utter_ask_location': 1.0}, {'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0, 'prev_action_listen': 1.0}, {'prev_action_weather': 1.0, 'intent_inform': 1.0, 'entity_location': 1.0, 'slot_location_0': 1.0}, {'slot_location_0': 1.0, 'prev_action_listen': 1.0, 'intent_goodbye': 1.0}, {'slot_location_0': 1.0, 'intent_goodbye': 1.0, 'prev_utter_goodbye': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_0_KerasPolicy
Predicted next action 'action_listen' with prob 0.92.
Action 'action_listen' ended with events '[]'
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Failed to run custom action 'action_weather'. Action server responded with a non 200 status code of 500. Make sure your action server properly runs actions and returns a 200 once the action is executed. Error: 500 Server Error: INTERNAL SERVER ERROR for url: http://localhost:5055/webhook
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Failed to run custom action 'action_weather'. Action server responded with a non 200 status code of 500. Make sure your action server properly runs actions and returns a 200 once the action is executed. Error: 500 Server Error: INTERNAL SERVER ERROR for url: http://localhost:5055/webhook
Encountered an exception while running action 'action_weather'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\keras\backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From C:\Users\meyss\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
